.github/workflows/oscal-cli-validate.yaml

```diff
@@ -0,0 +1,29 @@
+name: Validate OSCAL files
+
+on:
+  push:
+  pull_request:
```

Comment:
In meinem PR ist `
on: [push, pull_request]`

Ich bin da leidenschaftslos, wie, aber ich denke wir sollten es einheitlich machen.

Ansonsten LGTM